tests/ztest/error_hook/src/main.c

```diff
@@ -49,11 +49,11 @@ static void trigger_fault_illeagl_instuction(void)
 	((void(*)(void))&a)();
 }
 
-static void trigger_fault_access_null(void)
+static void trigger_fault_access(void)
 {
-	void *a = NULL;
+	void *a = (void *)0xFFFFFFFF;
 
-	/* access a null of address */
+	/* access a illeagal address */
 	int b = *((int *)a);
```

Comment:
We need to have volatile memory access here, otherwise it may be optimize out by compiler.

Reply:
Got it fixed, thanks very much for reviewing it and correcting on this.